tools/quantp/quantp.xml

```diff
@@ -1,11 +1,13 @@
-<tool id="quantp" name="QuanTP" version="1.0.0">
+<tool id="quantp" name="QuanTP" version="1.1.0">
```

Comment:
The only thing that I find strange is that with so many changes in the R script, the test does not need to be changed. But I guess this is due to the limited scope of the current test. Do you think the current test is enough to cover the functionality of this tool?

Reply:
No. I added another criterion to cover the extra functionality [here](https://github.com/galaxyproteomics/tools-galaxyp/pull/332/commits/060bf1c2422453f07a80346ce6e89341f031c715).